Makefile

```diff
@@ -39,7 +39,7 @@ generate:
 test: unit e2e
 
 .PHONY: create delete upgrade
-create:
+create: monitor
```

Comment:
This doesn't feel right.  I think we want the thing calling create/e2e to be able to start and stop the monitor themselves to wrap the whole create+e2e process.  The caller can build monitor, execute it in the background, record the pid, then kill the pid themselves on an exit trap.

Reply:
This is going to be fiddly to do short term because we want this to bridge the setup and test containers in the release template.  Just like with the upgrade work, I think we'll need to expedite moving template functionality to this repo for this.  I'd start with the non-bushslicer, non-conformance stuff.

Reply:
idea is that both `upgrade/update/e2e` is calling `make create`. but we will have to duplicate code inside in multiple places. In example scenarios:
1. Create is created, and we need metrics from creating only. Code will be in `create.sh` and it will manage it same as it is managing fakeRP process. 
2. Upgrade test is called. Upgrade calls `create.sh`, so now monitor script cant be initiated as part of `create.sh` because the script is being terminated as part of the upgrade script. So it invalidated scenario 1 as now logic needs to be more complex.
3. Update e2e is called. It same uses `create.sh` the script, so same logic applies. 


Im not very big fan of how we already managing fakeRP and killing it cia trap based on port `8080`. My thinking was to build in `timeout` in the process as housekeeping only. Traps would be used to send `sigint` signal into the pid.
And we call the process from `make` file as a dependency. In this case `make create` will call target `monitor` each time, and we just need to make sure it is cleaned after with another target `make monitor-clean`.
This way we can do for scenario 1:
`make create` -> `create: monitor init-cluster monitor-clean`
Scenario 2:
`make upgrade` - > `upgrade:` monitor init-cluster upgrage-init monitor-clean`